src/main.py

```diff
@@ -1,9 +1,10 @@
 from pathlib import Path
 from split_files import file_name_to_dict, convert_dicts_to_csv
+from parser import parse
 
 
 def list_files(location="./Photos/"):
-    return list(Path(location).rglob("*.jpg"))
+    return list(file.name for file in Path(location).rglob("*.jpg"))
```

Comment:
This is the main fix here. Instead of returning `Posixpath('full/path/name.jpg')` this returns `Posixpath('name.jpg')`